pkg/job/parser.go

```diff
@@ -64,6 +65,24 @@ func ParseStorageString(sourceURI, destinationPath string, options map[string]st
 			StorageSource: model.StorageSourceLocalDirectory,
 			SourcePath:    filepath.Join(parsedURI.Host, parsedURI.Path),
 		}
+		for key, value := range options {
+			switch key {
+			case "ro", "read-only", "read_only", "readonly":
+				readonly, parseErr := strconv.ParseBool(value)
+				if parseErr != nil {
+					return model.StorageSpec{}, fmt.Errorf("failed to parse read-only option: %s", parseErr)
```

Comment:
nit: user `%w` instead of `%s` to wrap the error. https://pkg.go.dev/fmt#Errorf (also below this line)